readme.md

```diff
@@ -167,7 +167,7 @@ We have a growing list of [common pitfalls](docs/08-common-pitfalls.md) you may
 
 Mocha requires you to use implicit globals like `describe` and `it` with the default interface (which most people use). It's not very opinionated and executes tests serially without process isolation, making it slow.
 
-Tape and tap are pretty good. AVA is highly inspired by their syntax. They too execute tests serially. Their default [TAP](https://testanything.org) output isn't very user-friendly though so you always end up using an external tap reporter.
+Tape and tap are pretty good. AVA is highly inspired by their syntax. They execute tests serially. Their default [TAP](https://testanything.org) output isn't very user-friendly, though so you always end up using an external tap reporter.
```

Comment:
`too` here serves to compare behavior with AVA.

No comma is necessary, but we'd add one it would be after `though`.